gravity-forms/gw-gravity-forms-rename-uploaded-files-usage.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+# First and Last Name
+new GW_Rename_Uploaded_Files( array(
+	'form_id'  => 628,
+	'field_id' => 3,
+	'template' => '{Name (First):1.3}-{Name (Last):1.6}-{filename}',
+	// most merge tags are supported, original file extension is preserved
```

Comment:
Looks like this got bumped down a line. Could we move it back inline? I think the context helps.

Reply:
Fixed!